docs/resources/metastore.md

```diff
@@ -52,6 +52,8 @@ The following arguments are required:
 * `name` - Name of metastore.
 * `storage_root` - Path on cloud storage account, where managed [databricks_table](table.md) are stored. Change forces creation of a new resource.
 * `owner` - (Optional) Username/groupname of Metastore owner.
+* `delta_sharing_enabled` - (Optional) Required along with `delta_sharing_recipient_token_lifetime_in_seconds`. Used to enable delta sharing on the metastore.
+* `delta_sharing_recipient_token_lifetime_in_seconds` - (Optional) Required along with `delta_sharing_enabled`. Used to set expiration duratrion in seconds on recipient data access tokens. Set to 0 for unlimited duration. The default value is 3600 seconds.
```

Comment:
@nkvuong from a resource end user perspective this field is set during the create portion of the lifecycle. Is this comment still relevant. As if the user sets this during creation it will still apply the patch after it is created. (in the code logic)

Reply:
https://github.com/databrickslabs/terraform-provider-databricks/blob/0647db04e9cd6b3811d4b38b57749db28a0eeb77/catalog/resource_metastore.go#L105-L108